plugins/modules/ansible_galaxy_install.py

```diff
@@ -188,6 +202,7 @@ class AnsibleGalaxyInstall(ModuleHelper):
     output_params = ('type', 'name', 'dest', 'requirements_file', 'force', 'no_deps')
     module = dict(
         argument_spec=dict(
+            state=dict(type='str', choices=['present', 'latest'], default="present"),
```

Comment:
Why use double quotes for the default, when using single quotes everywhere else in this line? :)

Reply:
Done ;)

Reply:
I don't care too much about which quotes you use, I just found it funny that you mixed them even in one line ;)